### DISCUSSION TITLE
Anti-aliasing

### DISCUSSION THREAD

**Discussion by dr-prodigy** [rds1983/FontStashSharp#12] [General]
Hi,
thank you again for your great work!

Since when I started using the lib, I noticed it applies anti-aliasing when rendering fonts, which is great for the vast majority of the cases.
In my case, dealing with lo-res screens (CGA CRT) too, I would appreciate also the possibility of disabling anti-aliasing at config level, just like you do for stroked or blurry, so to better match the capabilities of those devices.

I don't know the internals of the lib so maybe this is not even possible, but I'm wondering: what about adding a FontSystem property to switch anti-aliasing on/off?

**Comment by rds1983** [comment]
The only way to disable antialiasing is to use StaticSpriteFont and to load fonts from AngelCode BMFont files.
See this for more details: https://github.com/rds1983/FontStashSharp/releases/tag/0.9.0.0

**Comment by rds1983** [comment]
As for DynamicSpriteFont, it uses stb_truetype internally to rasterize glyphs. I couldn't figure out how to turn the antialiasing off there.

**Comment by rds1983** [comment]
Finally, there's a way to get rid of antialiasing: https://github.com/FontStashSharp/FontStashSharp/releases/tag/1.3.4
